addon/components/vertical-collection/component.js

```diff
@@ -230,51 +226,15 @@ const VerticalCollection = Component.extend({
 
   // –––––––––––––– Setup/Teardown
   didInsertElement() {
-    const containerSelector = this.get('containerSelector');
-
-    if (containerSelector === 'body') {
-      this._scrollContainer = Container;
-    } else {
-      this._scrollContainer = containerSelector ? closestElement(this.element.parentNode, containerSelector) : this.element.parentNode;
-    }
-
-    // Initialize the Radar and set the scroll state
-    this._radar.itemContainer = this.element;
-    this._radar.scrollContainer = this._scrollContainer;
-
     this.schedule('sync', () => {
-      this._initializeEventHandlers();
       this._radar.start();
     });
   },
 
-  _initializeEventHandlers() {
-    this._scrollHandler = ({ top }) => {
-      if (Math.abs(this._lastEarthquake - top) > this._radar._estimateHeight / 2) {
-        this._radar.scheduleUpdate();
-        this._lastEarthquake = top;
-      }
-    };
-
-    this._resizeHandler = () => {
-      this._radar.scheduleUpdate();
-    };
-
-    addScrollHandler(this._scrollContainer, this._scrollHandler);
-    Container.addEventListener('resize', this._resizeHandler);
```

Comment:
@pzuraq I think we have a bug related to this removal

basically we have responsive elements in the list that get shorter when a break point is hit. As a result in certain cases after resizing there are not enough rendered items to fill the scroll container. It fixes once you scroll.

I read through the PR but don't see any mention of this removal.  Is there a path forward to add our own resize handler?

Reply:
I found a path forward the other night.

Cc @pzuraq

Reply:
Ok, let me know if I can help. We are going to revert to `beta.8` for now.